public/components/ContentPanel/index.scss

```diff
@@ -1,18 +1,3 @@
-/*
```

Comment:
Add new copyright header after removing this old one?

Reply:
added